lib/createConfig.js

```diff
@@ -1,7 +1,28 @@
 const { merge } = require('webpack-merge');
+const path = require('path');
 const getBaseConfig = require('./getBaseConfig');
+const presets = require('./presets');
 
 module.exports = (commandName, configFragment = {}) => {
   const baseConfig = getBaseConfig(commandName);
+
+  if (commandName === 'webpack-prod') {
```

Comment:
It's because some MFE's have `webpack.prod.config.js` if I make the change in that file I would have to rewrite that file per MFE. Maybe I could be wrong I'm not sure. For the MFE's that don't have implemented `@edx/frontend-build` are a quite different because they use a make file any advise for that?

Reply:
The repositories that have a custom `webpack.prod.config.js` still rely on the base config defined in `@edx/frontend-build`'s `webpack.prod.config.js` file here: https://github.com/openedx/frontend-build/blob/master/config/webpack.prod.config.js

For example, [frontend-app-learner-portal-enterprise](https://github.com/openedx/frontend-app-learner-portal-enterprise), doesn't specify a `webpack.prod.config.js` file; instead, it relies on the default Webpack configuration provided by `@edx/frontend-build` as shown here:

```shell
❯ npm run build

> frontend-app-learner-portal-enterprise@0.1.0 build
> fedx-scripts webpack

Running with resolved config:
/Users/astankiewicz/Desktop/edx/frontend-app-learner-portal-enterprise/node_modules/@edx/frontend-build/config/webpack.prod.config.js
```

The `createConfig` function used in those custom `webpack.prod.config.js` files extend/override the `webpack.prod.config.js` from `@edx/frontend-build`. If you change the Webpack configuration file itself, `createConfig` would still inherit the change unless consumers explicitly override/replace the updated `cache` configuration you're adding.

The benefit being that consumers of `@edx/frontend-build` that don't explicitly use `createConfig` (like frontend-app-learner-portal-enterprise) would still get the significant performance improvements from the `cache` addition.

> For the MFE's that don't have implemented @edx/frontend-build are a quite different because they use a make file any advise for that?

[clarification] What MFEs in Open edX don't rely on `@edx/frontend-build`? `@edx/frontend-build` is the standard for Open edX MFE tooling & build scripts. Are you referring to non-MFE repositories that still rely on Webpack?

Reply:
@johnvente Bump on this comment thread 😄 I still feel we should be modifying the base webpack.prod.config.js file, not the `createConfig` function.

Reply:
@johnvente Just wanted to check in with you on this PR comment again. Thanks!